readthedocs/doc_builder/base.py

```diff
@@ -60,27 +49,8 @@ def build(self):
         """Do the actual building of the documentation."""
         raise NotImplementedError
 
-    def move(self, **__):
-        """Move the generated documentation to its artifact directory."""
-        if os.path.exists(self.old_artifact_path):
-            if os.path.exists(self.target):
-                safe_rmtree(self.target)
-            log.debug('Copying output type on the local filesystem.', output_type=self.type)
-            log.debug('Ignoring patterns.', patterns=self.ignore_patterns)
-            safe_copytree(
-                self.old_artifact_path,
-                self.target,
-                ignore=shutil.ignore_patterns(*self.ignore_patterns),
-            )
-        else:
-            log.warning('Not moving docs because the build dir is unknown.')
-
-    def clean(self, **__):
-        """Clean the path where documentation will be built."""
-        # NOTE: this shouldn't be needed. We are always CLEAN_AFTER_BUILD now
-        if os.path.exists(self.old_artifact_path):
-            safe_rmtree(self.old_artifact_path)
-            log.info('Removing old artifact path.', path=self.old_artifact_path)
+    def _post_build(self):
+        """Execute extra steps (e.g. create ZIP, rename PDF, etc) after building if required."""
```

Comment:
Does Python support empty function bodies?! 

I generally still think it's cleaner to have a `pass` or `raise NotImplementedError` here to be more explicit.

Reply:
He, yes. It does. I usually add a `pass` as well. However, it seems that Black removed it. We cannot have a `raise NotImplementedError` because it's always called by the builder and some of the subclass don't implement it.

I'm not sure if there is a way to tell Black that I want to keep the `pass` in this line. That's probably what we need here.